Automattic/wp-calypso
# Media Library: Autofocus Pexels Search Bar

#### Changes proposed in this Pull Request

* Ensures that the search bar in Media autofocuses for Pexels, since the action requested of the user is to enter something in the Search bar. 

#### Testing instructions

1. Visit `/media/site`
2. Click the top left icon on the bar
3. Select "Pexels free photos"
4. Verify the search bar autofocuses

<img width="1659" alt="Screenshot 2020-11-22 at 13 35 12" src="https://user-images.githubusercontent.com/43215253/99905238-8f58af00-2cc7-11eb-9d2a-264ed4e16f74.png">

cc @aaronyan, @johngodley, @gwwar 

Fixes #47397
